static/css/main.css

```diff
@@ -1,57 +1,107 @@
 
-/*Sidebar CSS - Base Template*/
+/* CSS for font-style*/
+body {
+    font-family: 'Fjalla One', sans-serif;
+}
+/* The side navigation menu */
 .sidebar {
-    width: 230px;
-    position: fixed;
-    z-index: 1;
-    background: grey;
-    overflow-x: hidden;
-    padding: 8px 0;
+  margin: 0;
+  padding: 0;
+  width: 300px;
+  background-color: #ff6600;
+  position: fixed;
+  height: 100%;
+  overflow: auto;
 }
+
+/* Sidebar links */
 .sidebar a {
-    padding: 6px 8px 6px 16px;
-    text-decoration: none;
-    font-size: 25px;
-    color: black;
-    display: block;
+  display: block;
+  color: #ffffff;
+  padding: 16px;
+  text-decoration: none;
+}
+
+/* Active/current link */
+.sidebar a.active {
+  background-color: #4CAF50;
+  color: white;
+}
+
+/* Links on mouse-over */
+.sidebar a:hover:not(.active) {
+  background-color: #ffffff;
+  color: #ff0000;
+}
+/*Header on sidebar*/
+.sidebar h1 {
+    font-size: 30px;
+    text-align: center;
+    padding-top: 25px;
+    font-family: 'Pacifico', cursive;;
+}
+/*Login form in sidebar*/
+.user_form {
+    padding: 10px 10px;
+    text-align: center;
+}
+
+.user_form legend {
+    margin: 0 auto;
 }
 
-.sidebar a:hover {
+.user_form input {
+    text-align: center;
+}
+/*CSS for styling user input fields*/
+.user_form input {
+    width: 70%;
+    display:inline-block;
+    padding:10px 10px;
+    line-height:10%;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+/*CSS style for submit button on login form*/
+.user_form button[type=submit] {
+    width: 60%;
+    background-color: #4CAF50;
     color: white;
+    padding: 10px 20px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
 }
 
-/*Sidebar CSS - Username/Password Template*/
-input[type=text], input[type=password] {
-  width: 100%;
-  padding: 12px 20px;
-  margin: 8px 0;
-  display: inline-block;
-  border: 1px solid #ccc;
-  box-sizing: border-box;
-}
-
-/*Sidebar CSS Submit button*/
-.button {
-    display       : inline-block;
-    border-radius : 20%;          
-    font-size     : 14pt;       
-    text-align    : center;      
-    cursor        : pointer;     
-    padding       : 12px 12px;   
-    background    : rgba(102, 102, 255, 0.84);     
-    color         : rgba(255, 255, 255, 0.72);     
-    line-height   : 1em;         
-    transition    : .3s;         
-    box-shadow    : 7px 7px 3px #666666;  
-    border        : 2px solid rgba(102, 102, 255, 0.84);    
-  }
+/* Page content. The value of the margin-left property should match the value of the sidebar's width property */
+div.content {
+  margin-left: 300px;
+  padding: 1px 16px;
+  height: 1000px;
```

Comment:
I did some testing of this CSS and noticed that we have a huge amount of whitespace with the change and an automatic scrollbar that makes some elements show up at the bottom of the page. I think the margin-left and height should be removed here. 

Additionally lines 120 & 127 should be updated from 245 to 300 in order to prevent overlap :)